src/frontend/src/app/problem/code-editor/code-editor.component.ts

```diff
@@ -1,20 +1,25 @@
-import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Component, Inject, Input } from '@angular/core';
 import { CodeModel } from '@ngstack/code-editor';
+import { SolutionSubmitService } from '../solution-submit/solution-submit.service';
+import { MatDialog, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import * as prettier from 'prettier/standalone';
+import * as tsParser from "prettier/parser-babel";
+
 
 @Component({
   selector: 'app-code-editor',
   templateUrl: './code-editor.component.html',
   styleUrls: ['./code-editor.component.scss'],
 })
 export class CodeEditorComponent {
+  @Input() id = '';
   theme = 'vs-dark';
 
   codeModel: CodeModel = {
-    language: 'typescript',
-    uri: 'solution.ts',
-    value: [`export class MyClass {`, `  constructor() {`, '', '  }', `}`].join(
-      '\n'
-    ),
+    language: 'js',
```

Comment:
```suggestion
    language: 'javascript',
    fileType: 'js'
```

Seems like the syntax isnt highlighted (see SS)